packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barrier-selector.jsx

```diff
@@ -0,0 +1,153 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { Localize, localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, InputField, Text, ThemedScrollbars } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import BarriersList from './barriers-list.jsx';
+
+const BarrierSelector = ({
+    barrier_1,
+    current_focus,
+    onChange,
+    setHoveredBarrier,
+    setCurrentFocus,
+    turbos_barrier_choices,
+    validation_errors,
+}) => {
+    const [selected_barrier, setSelectedBarrier] = React.useState(barrier_1);
+    const [is_barriers_table_expanded, setIsBarriersTableExpanded] = React.useState(false);
+
+    const header_tooltip_text = (
+        <React.Fragment>
+            <p>
```

Comment:
@kate-deriv let's use <Popover component for the tooltip and <Text component for the text inside it. Please check an example for it in packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/info.jsx🙂

Reply:
Done, thank you for advise